### DISCUSSION TITLE
Build Failed: Command "pnpm install" exited with 1

### DISCUSSION THREAD

**Discussion by tinyramenstudio** [vercel/vercel#10794] [General]
### Description

Hi, out of nowhere I can't deploy my Nuxt 3 to Vercel anymore.

It says `Command "pnpm install" exited with 1`, but no further details given. I also can't see the Deployment Details -> Building which usually provides console information about the build.

<img width="1267" alt="image" src="https://github.com/vercel/vercel/assets/139680665/e22b190d-97ab-4df4-bc67-eef44d7364a0">

**Comment by bennobuilder** [comment]
Could it be related to this issue? 
https://github.com/vercel/vercel/issues/8272

Without further logs its hard to tell what went wrong..

**Comment by antoinebonpro** [comment]
i have the same problème with no permission
